lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexDriver.java

```diff
@@ -76,7 +79,9 @@ private VortexDriver(final EvaluatorRequestor evaluatorRequestor,
                        @Parameter(VortexMasterConf.WorkerMem.class) final int workerMem,
                        @Parameter(VortexMasterConf.WorkerNum.class) final int workerNum,
                        @Parameter(VortexMasterConf.WorkerCores.class) final int workerCores,
-                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads) {
+                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads,
+                       @Parameter(VortexMasterConf.InputCodec.class) final Codec inputCodec,
```

Comment:
I've found that when a Tasklet finishes, we need to ship the output codec for deserializing the output. It means `VortexFunction` should be added to the `TaskletReport`, and deserialize the function first to get the codec. @bgchun Would this be okay? 

Or, if we go with [REEF-1003](https://issues.apache.org/jira/browse/REEF-1003)), this can be done by instantiating the `VortexFunction`. If this is the case, could we resolve that issue first?